extensions/applicationinsights-react-native/package.json

```diff
@@ -3,33 +3,43 @@
     "version": "2.3.5",
     "description": "Microsoft Application Insights React Native Plugin",
     "main": "dist-esm/index.js",
-    "types": "dist-esm/index.d.ts",
+    "types": "types/index.d.ts",
```

Comment:
Changing so all packages are dropping files in the same way -- package.json for project also updated to point to this new location for types.